cos-nvidia-gpu-installer/installer.sh

```diff
@@ -48,7 +48,7 @@ BIN_OUTPUT_DIR="${BASE_DIR}/bin"
 KERNEL_SRC_DIR="/lakitu-kernel"
 
 NVIDIA_DRIVER_DIR="/nvidia"
-NVIDIA_DRIVER_VERSION="375.26"
+NVIDIA_DRIVER_VERSION="375.51"
```

Comment:
You will need to update the checksum below.

Reply:
Done